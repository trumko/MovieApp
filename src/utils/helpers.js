export const getMovieYear = (releaseDate) => (
  new Date(releaseDate).getFullYear()
)

export const shuffle = (array) => (
  array.sort(() => Math.random() - 0.5)
)
