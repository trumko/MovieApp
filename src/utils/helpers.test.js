import { getMovieYear, shuffle } from './helpers';

describe('getMovieYear', () => {
  it('should properly extract the year from 2018-03-30', () => {
    expect(getMovieYear('2018-03-30')).toMatchSnapshot();
  });
  it('should properly extract the year from 2019-01-31', () => {
    expect(getMovieYear('2019-01-31')).toMatchSnapshot();
  });
})

describe('shuffle', () => {
  it('should return the array', () => {
    expect(Array.isArray(shuffle([1, 2, 3]))).toBe(true);
  });
})
