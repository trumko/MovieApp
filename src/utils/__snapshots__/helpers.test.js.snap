// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getMovieYear should properly extract the year from 2018-03-30 1`] = `2018`;

exports[`getMovieYear should properly extract the year from 2019-01-31 1`] = `2019`;
